internal/accounts/create.go

```diff
@@ -286,6 +281,58 @@ func createInteractive(state *flowkit.State, loader flowkit.ReaderWriter) (*flow
 	return onChainAccount, nil
 }
 
+func createNewAccount(network, publicKey string) error {
```

Comment:
this should be extracted elswhere. I suggest creating a "gateway" module outside of flowkit in the internal module. In there we should move all the APIs implementation (mixpanel, sentry, this one). Then we just call those methods where needed.